backend/src/main/java/ai/verta/modeldb/versioning/BlobDAORdbImpl.java

```diff
@@ -1298,16 +1293,31 @@ private String getStringFromLocationList(List<String> locationList) {
   /**
    * This method find the blobs supported based on the following conditions
    *
-   * <p>commit.author, VERSIONING_COMMIT.label, VERSIONING_REPO_COMMIT_BLOB.label,
+   * <p>commit.author, commit.label, VERSIONING_REPO_COMMIT_BLOB.label,
    * VERSIONING_REPO_COMMIT.label, repoIds, commitHashList
    *
    * @param session :hibernate session
    * @param request : FindRepositoriesBlobs request
    * @param currentLoginUserInfo : current login userInfo
    * @return {@link Map} : "result", "count" as a key
    */
-  private Map<String, Object> getRootShaListByCommitsOrRepos(
-      Session session, FindRepositoriesBlobs request, UserInfo currentLoginUserInfo) {
+  /*private Map<String, Object> getRootShaListByCommitsOrRepos(
```

Comment:
remove and move comment to the destination location

Reply:
Done